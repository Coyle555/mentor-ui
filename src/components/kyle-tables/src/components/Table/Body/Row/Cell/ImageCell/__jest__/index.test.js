import React from 'react';
import { ImageCell } from '../index';
import renderer from 'react-test-renderer';
import { cleanup, fireEvent, render } from '@testing-library/react';

afterEach(cleanup);

test('Default render of image cell', () => {
	const tree = renderer.create(<ImageCell value="/foo" />).toJSON();

	expect(tree).toMatchSnapshot();
});

test('Render of image cell with no value', () => {
	const tree = renderer.create(<ImageCell value="" />).toJSON();

	expect(tree).toMatchSnapshot();
});
