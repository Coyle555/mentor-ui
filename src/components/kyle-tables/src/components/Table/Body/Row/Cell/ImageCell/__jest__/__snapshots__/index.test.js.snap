// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Default render of image cell 1`] = `
<img
  className="table-image-cell"
  src="/foo"
/>
`;

exports[`Render of image cell with no value 1`] = `""`;
